### PR TITLE
PMM-12892 exporter test for gh actions

### PR DESCRIPTION
**Pull request by yurkovychv**
https://jira.percona.com/browse/PMM-0

- [ ] Links to relevant pull requests

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3536-964eed5
client docker - perconalab/pmm-client-fb:PR-3536-964eed5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3536-964eed5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3536-964eed5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3536-964eed5.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14886/

File: .github/workflows/pmm2-ui-tests-fb.yml [hunk 1]
```diff
@@ -106,6 +106,20 @@ jobs:
       #      tags_for_tests: '@bm-common|@bm-locations' -- include bm-common once setup for mysql fixed
       tags_for_tests: '@bm-locations'
 
+  exporters:
+    name: Exporters tests
```
Comment (saikumar-vs) [inline .github/workflows/pmm2-ui-tests-fb.yml:110]: Should we be explicit its only Mongo Exporters?

**Review by saikumar-vs — APPROVED** [review]
Thanks for workingon this, one comment pls have a look.

File: .github/workflows/pmm2-ui-tests-fb.yml (hunk 1 above)
Comment (yurkovychv) [reply .github/workflows/pmm2-ui-tests-fb.yml:110]: Nope, it also includes other exporters tests

**Review by saikumar-vs — APPROVED** [review]

**Pull request merged**